c2pa_c_ffi/src/c_api.rs

```diff
@@ -586,7 +586,7 @@ pub unsafe extern "C" fn c2pa_reader_free(reader_ptr: *mut C2paReader) {
 /// Returns a JSON string generated from a C2paReader.
 ///
 /// # Safety
-/// The returned value MUST be released by calling c2pa_string_free
+/// The returned value MUST be released by calling c2pa_string_free,
 /// and it is no longer valid after that call.
 #[no_mangle]
 pub unsafe extern "C" fn c2pa_reader_json(reader_ptr: *mut C2paReader) -> *mut c_char {
```

Comment:
The real bug here is that the Json string is not being treated as a constant. We should change this!